### PR TITLE
Readme portuguese translation

### PR DESCRIPTION
**Pull request by jamellyf**
Hi, this pull request aims to add Portuguese translation to the file _README.md_. I hope the community approves my contributions.

### CODE REVIEW

**Comment by RedSoxFan** [comment]
Related #3166

File: README.pt.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# sway
+
+O sway é um compositor do [Wayland](http://wayland.freedesktop.org/) compatível com o [i3](https://i3wm.org/).
+<!-- sway is an [i3](https://i3wm.org/)-compatible [Wayland](http://wayland.freedesktop.org/) compositor. -->
```
Comment (lovesegfault) [inline README.pt.md]: Maybe remove these?

Comment (jamellyf) [reply README.pt.md]: You mean remove the comments?

Comment (lovesegfault) [reply README.pt.md]: Yeah, is it expected that they remain or were they there just to aid translation?

Comment (jamellyf) [reply README.pt.md]: Yes, I just kept those comments to help me with the translation.

Comment (jamellyf) [reply README.pt.md]: Should I remove those?

Comment (ddevault) [reply README.pt.md]: Yes, please.

Comment (jamellyf) [reply README.pt.md]: I removed them.

**Comment by jamellyf** [comment]
I don't know why, but now the building process is failing.

**Comment by ddevault** [comment]
Can you rebase against master?

**Comment by jamellyf** [comment]
> Can you rebase against master?

You mean checkout to my_branch and do `git rebase master`?

**Comment by emersion** [comment]
See https://git-rebase.io

```sh
# On your branch
git fetch
git rebase origin/master
```

**Comment by jamellyf** [comment]
> See https://git-rebase.io
> 
> ```shell
> # On your branch
> git fetch
> git rebase origin/master
> ```

That outputs "Current branch readme_portuguese_translation is up to date."

**Pull request closed without merging**